src/Domains/Test/UnitTestGenerators/NotificationTestGenerator.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Soyhuce\Somake\Domains\Test\UnitTestGenerators;
+
+use Illuminate\Support\Str;
+use ReflectionClass;
+use Soyhuce\Somake\Contracts\UnitTestGenerator;
+
+/**
+ * @implements \Soyhuce\Somake\Contracts\UnitTestGenerator<mixed>
+ */
+class NotificationTestGenerator implements UnitTestGenerator
+{
+    public static function shouldHandle(string $class): bool
+    {
+        if (!class_exists($class)) {
+            return false;
+        }
+
+        if (!Str::is('Domain\\*\\Notifications\\*', $class)) {
+            return false;
+        }
+
+        $reflectionClass = new ReflectionClass($class);
+
+        return $reflectionClass->hasMethod('toMail');
```

Comment:
Simply check if $class extends Illuminate\Notifications\Notification

Reply:
Done in f765190a76e70008ddcf5b00724935b2c5825fd2